include/mbedtls/config_psa.h

```diff
@@ -66,6 +80,13 @@ extern "C" {
 #endif /* !MBEDTLS_PSA_ACCEL_ALG_ECDSA */
 #endif /* PSA_WANT_ALG_ECDSA */
 
+#if defined(PSA_WANT_ALG_GCM)
+#if !defined(MBEDTLS_PSA_ACCEL_ALG_GCM)
```

Comment:
Ditto, as for CCM

Reply:
This block of code was accidentally left in place in [ab5631](https://github.com/ARMmbed/mbedtls/commit/ab5631fbee6234b9e47479b864d46f6b73cd31c3).

Please remove it.